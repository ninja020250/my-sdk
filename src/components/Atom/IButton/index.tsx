import React from "react";
import { Button, ButtonProps } from "@chakra-ui/react";

export type IButtonProps = {
  children?: React.ReactNode;
} & ButtonProps;

const IButton = ({ children, ...rest }: IButtonProps) => {
  return <Button {...rest}>{children}</Button>;
};

export default IButton;
