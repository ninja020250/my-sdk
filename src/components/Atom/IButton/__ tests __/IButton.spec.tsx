import renderer from "react-test-renderer";
import IButton from "..";

it("renders correctly", () => {
  const tree = renderer.create(<IButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
