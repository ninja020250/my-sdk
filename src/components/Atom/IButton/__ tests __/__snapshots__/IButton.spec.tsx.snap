// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="chakra-button css-4xx2wk"
  type="button"
/>
`;
